android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.2'
-  compile "com.google.android.gms:play-services-base:9.2.0"
-  compile 'com.google.android.gms:play-services-maps:9.2.0'
+  compile 'com.facebook.react:react-native:+'
+  compile "com.google.android.gms:play-services-base:8.4.0"
+  compile 'com.google.android.gms:play-services-maps:8.4.0'
```

Comment:
I only have a problem with the version hard coding because it means `react-native-maps` is broken out of the box 3-5 days every 2 weeks.  In spirit, i agree with your take on "know thy dependencies".

Something I've always looked up to about this project was your README file that said, "Look, RN is new and unstable.  Stay up to date.  Here: have a * in our package.json". 

I just hate to see you guys doing grunt work when new releases come out.  :)

RE the 9.2 :  If you think that's the right thing to do, I guess we'll just have to have people upgrade.  I don't know of the bugs you're talking about, but you have a bazillion times more exposure to that than I do.  :)

Thanks again for considering these changes.